.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - 'stable'
+before_install:
+  - npm install sequelize-cli -g 
+install: 
+  - npm install
+services:
+  - postgresql
+before_script:
+  - npm add sequelize-cli
+  - psql -c 'CREATE database bn_test;' -U postgres
+script: 
+  - npm test
+after_success: 
+  - npm run coverage
+  
```

Comment:
A blank line at the end shouldn't have space characters. Please make sure you have removed them
```suggestion

```

Reply:
@kcharles52 Thank you, space removed